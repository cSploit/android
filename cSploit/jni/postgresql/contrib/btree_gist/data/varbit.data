10101001001110001110101100110110000000110011010000101000100010001000011110000111100110111111111
001110101100110
010000011101100110001001111111101110011110110100011001101011110010101100111011011101101101
1111000010000011110011101001101101011110101111011011001000000101111110000000111011010100011001
1111110111111000000100000101010010011101111100010100010101101101110010111110110010110000000000
000000110100001010001011101110111110100111010011000101111111000100001001000011001011000111101
0100001010001011101110111110100111010011000101111111000100001001000011001011000
011000001000011110010000111001110101000110000011000000011100010111000111001010111110
1010100011001111110111111000000100000101010010
01001110100110001011111110001000010010000110010110001111
1011000111101100000110100000010101010111101100000100001
01011011001100110100000011000101000100010100100000110100011110010001010000011
0111101100000100001111001000011100111010100011000
1101101110010111110110010110000000
11010111101011110110110
01000001111001110100
01101101011110101111
1111100110010111101111110101010110000010001111000000110010101010011100100011011000001101011
1010000111011010111001111111110000001011010011110101101100100000010111010111
101011011100000010111000010011011100000
011111000000001101111010000111011001100001110101001110
11
\N
0010101010011100100011011000001101011010110001101010110010110101001000000
0101111101011011001100110100000011000
110111
1001111001010101110101000100110110110011000111000011011110101010001110011000100011101
110000001011010011110101101100100000010111010111101100
0001010100101110001001000010001101010010001101001001010111100011101101100001100000011
101000000101010101111
00011101001010111110100000101011000110100101100001111010111110110001000111101100011011111000000
1001001111001011100001010100101110001001000010001101010010001101001001010111
1100011101101100001100000011110111110100011000110100010101111111011011011010
0010101110100110001001001011011111111111101011
0100111010011000
1111101000110111011011111101001010011111001000110011101010000100101110110101010010101001001
11110
00011011100110110001000000010101111111111011101111010
011011110100001110110011000011
01111010111110110001000111101100011011111000000101100001100000001101000001
111101111111100111111011000110000100111100101010111010
001
1110100111010
01001101001111101001000000000000100100101010110001111101111100
\N
\N
0001100101101111000100000000101000110100000100101001001010000100000001000001011010
10110110111110100001010011000001010110001101000011101010001101101100111100010011000111101111001
11010101101110101001110010111111010101101000100000011110
101001000000000000100100101010110001111101111100111000111
000011001101101011110101000110001101011000
01100001111100011100000110001100101111100100111101100010101101101111101000010100110000
01011111110110010010010000000000110111000010011011111000001100100001010001100100010101010111110110
111110101010000111111111110000010001110001010000011110001110011001111010101111000
00011000001000101111000000001000010000011100011010100100101100011110101
010101111011
11111110111000010111101110101011100
1110011001101011010000100010101110110111000110100010000011010101011101001011000011010111101010101
0000001111011010011010110010100001000010011010011110111011101100000101110011111
00101001000011010110100100010111000101011100110000110111100001010010011
100101000010000000100000101101010100000110000010001011110000000
010110001000110010110111001101011111110000101100110010000010011001011111101100010110101011001
01011110001110101110000011100011111111011111010000
110101100001100110110101111010100
010000010100111010111010000100000110011100011110110011
1001100101111110110001011010101100110110101100100100000100101101110001001101111001101010
010101110011001010110100000111
\N
0100011010101110110000010011
011101111101000110111011011111101001010011111001
10010100110110011011001111110001001000110111101011101011010110001001001111110
1110111110011100011111101010110001101110
01010101110101000100110110110011
10101110100
001110100000100101000001010011110101110111011000100000
100000111010111111110011000100100000111100000010100100011011101001110100001000100011
101101000001001010011100001111000110000100100011100001000111111011110010110010111101
11111011111001001110111111101
1111111011100001001100100001110100011010110000000001001010101001001111101000011110011000000101111000
001111000110000100100011100001000111111011110010110010111101001100010101110001110
011000001000101111000000001000010000011100011010
110010100001000010011010011110111011101100000101110011111
000011101101110001
01111001100111010000100100110101
000001010001100010010010001001000101111000011111
1001001011011111111
01100111000110001010010000100101011101010
11001011111111011111111111111011100001001100100001110100011010110000000001001010101001001
1101111010111100011
\N
00101011000110100001110101000110110110011110001001
001001101110000010100011110011110101111101110100000101111111101011101100111010101011111010
010101111001101011011111011111000101110101000110010010010010111010001100011001011011100110011000001
0000011100
00110010110111010100111111011111111111101100110001000001101111010000100011101110101100010101
110100110010001011000010010101100010001100101101110011010111
000111000111111110111110100000010
0000001011010011
00100101000001010011110101110111011000100
\N
001000110110000011
1101100110110010
001101011101001100010101
0001001000101111000011111110011100110011
10101010101000100101101001111011101000010001101
000001011001100010011110010011101010001011000110110001000101110110001100001001110101
10011000011101010001010111101001100011100100001100011100001011110011
110111110100001010011000001010110001101000
000110000010001011110000000010000100
0100000100101100101000101100010000010011110111001100000000000111
000000111001010001001111101110011011011110011111100011101001001010010011100011111100100010
001000011111011010000110110111101010000011111100010000110111111110
001
000011110001101001110111110101101101111110111110001001001100011011010101010100101101101011
00011010101100111110001110011001011111111010111010110011101111000100101010
1011011001001000110010100000111001000111010101
00110100000010111010100110001101100011001110101011101110111111010000000100010100010110111011
1101000001010111110110111001011111010000110000110010101111101010100001001000110
101111000010100010100010000000010110011000001100011010101001011011110110110000110
00100100001101100011101111110011101110000001001111
010011010001000001110011101100001111011011110101111
\N
\N
1001010010011100011111100100010100100110011110111111100101110110101111100011111111
001111110000001100111011001010010100000000110110001110100100010110000010011011110100011011
11110110011100011000101001000
00101001110110111111101011100110110011110110000101100010100110
011111010101001011000011111000000000101101100101101111100011110011
10101000110001000001110101111111100110001001000001
0010100100011010011100100011111110001101010000011101110100101101011
111111001111110110
1111110101010011101000111
001011001111101010000011001101110001011011000011100
0000001111010111000100111000111010010001100010010101001101101110010101001101000001
011
00000010110010
000000111111110101010011
101
0100
001101101100111100010011000111101
111000010100111101010111010111010011000010001001100010001110001000000111000010101111010011101110
000101100110010110101100001000100000000011010101100111110001
10001000111011001101000110000101100011000000
11101010000011001101110001011011000011100001101
\N
0000001101011110001010101010101000
11010011110100101011110110101110001101011101110010110001000010010011011010011
1010010100000000110110001110100100010110000010011011110100011011010011110100
10110011110110000101100010100110110111011110110010110
1001111001010101
100011000010110001100000001100001110111100
\N
001000001110110111111100011000011011010101
001100101
000101001100010101100011000101111011110000101000101000100000000
0000110010101111101010100001001000110101000110110101101000010
1011000011010111101010101011011100000
1110100101010010100101100101101111110111000110101100
001001011000000000000101011100110111000
0101111111000010110011001000001001100
01001110110000111111011100011001101001010100000100011101110010101010011001010100
0001100011111110100101100111110010000111110110
111000100000000101000110
01011010101101110011001100100110100010010100011010000111101000000
001111010100000010101101100010001111100100
11111110011111111010101011000101111111100110010110111
01101010001000111111010
00001110111100010000011101101111111000
0011101000101011100011100000
11000111111111011010100110011100110011100
10100010
101010001101101100111100010
1001001011
\N
11000110
01100001110000011110001101001110111110101101101111110111110
0101000110010011001110110111100010101011001101000000001111110111111111100101100001100110101
1111000
100011001011011100110101111111000
0011
0001000001011010101000
10001110100100010110000010011011110100011011010011110100101011
111000001110001111
11001100011101001110010110100111000101011000000001111010010001
1111100011110011000111101111110011000000010011
010010001101111010111010110101
10100000110111100000010
10010111110001010001110001001001111000001100010011001101110010111000101000001001000
1011110100101001100011111011100100011011010010011110000001010110
\N
000111
0011000010100110001011000000001110110111110001101100010110100101001101100100001100001101110
1010110001
10110
110011000001111110101011101100111110011111111111101001010110101000101100011001010111111000
100000100001101011011111011000001011111000000001000011100110100010101
111001011001011000010100010010101010110110101101110000000101110
100111001010100011011101110100110101011111000000111011011010010011111110011001000110000010011110
1011111000010111111010100100101010000101101110011110001000000110000001111010001100000111101100000110
0011001010111111000100010101001001100110010100101001011011100010000011110110101000000001
01110
11111011011110001000111100101111001
0010011110010010111001100101111011110001101011011011011010010000
101101101111110111110001001001100011011010101010100
001100111000111101
1001010001001111101110011011011110011111100011101
11000111111100110000110101100101100
0001101101010100011100100100001
000111100000110110101111001101110101100001000101111001101000011010
1110001010010000001001110011101101110101011110000100000110110111111010000101110000001001100111100
0010000010110001011001001011110101110100101011001111011100110000101111011110100
00111001001111101011101111001011110100001101100011101001100010110001100
110101101100100001011000111001011011011010100100011101110000001011010110011011011010111110
110000111100101111010010100110001100001010011000101100000000111011011111000110110
000001011111000111100101001001010111000101010101100001101101100010101000101011110111111
00100010100100110011110111111100101110110101111
1011110011001
10000101110100100101011010100001010010010011100000001100011000000100100101101101100
1011111110011011011001100001110010011111100111101000100011010010000111100000001101001010
110011011011100000100110000011010000111101100110001100101011
100000110101101111000011100000001110010001100111110101101011011000010000111001101011110011
10100001010110100100111000001111001110011011101110
110100100101
0010100110100010000011100111
10101010010100000111010101000001111101000101101
001000001010001111110011000011010000
0100111110100
11111111011111111111111011100001001
00101010000011010111111011000011000101011100001101011111111111001001110100000000
110111101011111000111010110100000
011001
11101111111011111011110010011100100101111101101110011111001101001001110
01100010010101010110111000110111100000000110011101010011100001001001000010001100111110100100010010010
\N
101011101111100101010000000000010100101000111110110001111000011001000001111000100011111111011
0010010110
11111110101111010100000000000110000001000010011001
010110000000011110100100010000110000001101100111
001000000111000010101111010011101110100100110011101010101
0111110011110011100001100010010011010111000110111100011101000
110001111010011000111111100110
111101010101011101
1001
10011110111110011001100011101001000101101
0100001001100111011110101111100
011010011100110100110011011110111100110111011011110101110110011000011111000010101011111001100111000
000011001110001
0001000010000111000111111000111111001000110010111011001001
111100101111111000101101001001001000001111101111010
00010000111001010011001111100111101010011010100
001111111010100111100010000
0011000111011111101001001100111010001101100101010101011111110101001
010110111001100110010011010001001010001101
1010001110
001011110010110110010100100101000000001010000010101110010010111111110001101011110000000001101
010010001100111110010000010011011110101011110101001111000000011100111
110001
10011110111111001100110100010110100100101000100100100111101000100101011101101111
0100111000010010010000100011001111101001000100100101100101111010000110011110100011100101000
01101100011100
011001111000100110
0001010011000101011000110001011110111
10011010100010011100110011101110000010001100011011100101101010010011001000
100111001000111111100011010100000
01011110000111101011011000110110111100111100001001100
101001011110010110100100100010110100101100011100110010101000111110110010001001111001011101010001
00000101011011000100011111
1110010000100001111101101
111100011001010001110
011011100011011110000000011001110101001110000100100100001000110011111010010001001001011
0001101111000000
011010000101011111010010101001
010000101001001001110000000110001100000010010010110110110001111101
0001111110110010110111000000111101010000110011011010100101101101110000001100111001101110000101000
0000011101110010010101
1001100001111100001010101111100110011100000001011100100000111111001110011011100001111101
1101001100010100111010100011011000110111101011011110
\N
0110100101010000010001110111001010101001100101
10010101011010011101011010011100010100001011010010111011110111001100011010110101010110110
00100010110011010101011001
111000100001001000010010000101011101010000011100110110111011011101100111010101010001110110100
1100000111101100000110000011010110111100001110000000111001000110011111
011110101
11000110001011100010100100000010011100111011011101010111100001000001101
0110110100101001001000001000011001100000101010101010000111100011010001101101000
11101001100000010100101111101110000101001111010
01100101100100001001100111101111000010100000001010001001001111011100010111010010001100
1010001000011000110011011000111111110001101010100110101011100100001001010011000001000101
10101011101011110010000110110101100110011111101
11000001111011000001100000110101101111000011100000001110010001100111
1111000100000111011011
001101100111111110000011011101111011011
00101
100100110000000110010010000001010111100011100001110000010000110000011111111011010101010010111101
0111010111101110010101101011011000001000000100001000000010110011000010100010111111110000001110
010011011111000001
01011001100100000100
11111011110101001101000110011101110111101010110111010101100001101011110101011011100101001000100101101
101000010100000000011000010000101001111001110100100001001100010010101010110111
1100011110010101010100111111011000100100100110101001100100110011110100001110011
1001100110011101110000011101110100000010001
000001110100010011101100000001001111110101110000101100100010000101110011101011101
0010000010011011110101011110101001111000000011100111001101
00
1110011100
11010110001010101111101001010
111000000010010101001111111111001011000110011010010111010000
10010110001000101110111110111001001101001001011010010110101001001111011
100010100010000110001100000110110011011101001101000101000110010001111010
01010001100001111011000001001111100100000100001101110001101010101011010110000001100111100111
00010010110111110100101001010100011011010100111010011100001101011100101000100000010110101100100010100
10010010000000000
011110101001101000110011101110111101010110111010101100001101011110101011011100101001000100101101
111100011001010001
11101111010111101101000001101100111111101110
01101110101000001111001010111001011110001000011111011000101011001011010000101001100001111101
10011
01100011011011110011110000100110001011100001
10111010000100011011111011100010011101111001000
00100111001011111001110110110000000110111010111001010010111011011111000
0001011111011010000001010011000001111011011010110010
10000101110110001001010001110110000010000000010000101110110111011010001010
010110110010000100001110011010111101101011001100010101
111010011111011111111100001110010100110101111010
011101010100111010001110000110100010011101010011000111000010000001011101100101001000001000
1100110101000101010011000010101011111010011111011000100000001
0100100011101110000001011010110011011011010111110001110100
1000110101100111110111101
001111001010011010101101110011101111010111
011101010100000100101010101111010000100010010110001000101110111110
011011001001001010011001100100100011001100001011100111101000101010001101000110001010111
100000001100011000000100100101101101100011111010110111
10010111011
101111000000001
001000100010111110011011011100000100110
1110111111101000111011011100001
110010011110111011011110101100111100011
110110110001100110011110001111111011000001110100010011101100000001001111
110100011010001001
11010101101000011011001110100010101010000110010010110
010110001001000110100100000000100011110011011111101000010111001110000101010000
0100000111001101101110110111011001110101010100011101101001010111001000100001
\N
000001111000100011111111011111010001101110100111110001011010111
000110000110111000100110101101011111001111
01001101011001
0011111110101010001000000101110010001000000110100000110010100100010100111001100111000
00111000111111001000101001001
10000110
111111100000000100100100001100
10000110011010000000000010100011110101101010011110001110100100110000000110010010
1001001011000111111011101000
111100011111010111101101101100011001100111100011111110110000011101000
1010101111000
011111
011010000011011001111111011101111010011
101011001111110101010011000001100111011000001110000011000101101101000001110000011110001010
01011101110010010100111000101000100001100011000001101100110111
0110000110011011110000010111110100111001010100011011
000101000111101011010100111100011101001001100000001100100100000010101111000111
1101011100001101001010110111000011100011101
0001010110001
110010001111111000110101
100000
\N
10110111001100110010011010001
001111111001011001100101000010011011001000111111011001011011100000011110
00000011100110000110000001010110111011101
1011100111101000000111000
01100010011101010111010111011111001011010001101110000011111000101100001000101011110000100100110
001000111011101011000101
10100001011111111001101000000100100100001000100011001100001100110001100000100100011100111110100
100111
00110010101000111110110010001001111001011101010001001011001111101011
1000111010110111111101000001010110110000010
11111010111010110011101
0100100001000001010000001000010
101101011000100
0001111010111111100000010111100100100101111000010000111
1001011100100110010000010101101010100101000100100011100
1101010000011101
110000010000011110011110001011010111100001001001
111101001101011111010110110000
010111010010110101101000101011110100000011100000011101101110
01111110111111111111011
0001011010101100001101101111000
0101010
001110010000001010111001101101110101111101111001010101111101110010111
101111010
000001010100110010110000011110000011010110100001011101010110111011101101010111010011100011010000
011100111110011010010011101100000110110110
0011011010111010110111001101000000110111100000010111000010101100101110001001001101001100100110000100
1001100000110111110100100100011000000001111100011001111110001111110010100
10010001100101
0011101110111101010110111010101100001101011110101011011100101001000100101101
011100011000001101111010001000000111011100111111000010010100100101
11
111000010010101101101001000010001000110001010111010000000111010101001110101
1000000100100110001101111001011100000111010010000001010100101110000010011101100111
01111000
0011011110100011011010011110
10001011101010001000010001011001011101100101011001100100111011101000000011101100100111100
001001100100100000001011100111001001000001110001110011110110110010010011101
1001101000111011110101110101001001000101000000010101011110111110
010
1100110101000101010011000010101011111010011111011
00011101011110110100100010110011110010000010111001111010000110010110000000
100001111010111111100000010111100100100101111000010
\N
10000111011101100011111010011011110010000100010100000111100001011010010001001100010110001001111
10110101011100101111011101001111011111011011101001110000010111100001110110111001011010111111111
1101110110110000011100010000000001101000101001111010
1010010101100110010000001010101000111010111101101001000101100111100100000
011011101101110110011101010101000111011010010101110010001000011
1111001011100000111010010000001010100101110000010011101100111000010001101100010
10
0100010100111011010110110000100010110111000101101011101111101010101011010
0111110001010001110001001001111000001
1101000101101001001010001001001001111010001001010
0111111101000100000111001001101111000011010100101010010101110110111100011
110110100010010110010010011110001010100111110000101100011110110011011111101110011111111
1010011101110010101000111100100011101100001110000
0111110000000000000100110110011010011011110110001
001011110101110100101011001111011100110
0011110010110111001010011111001111110001
1000110010111001101000111100111111001000000000111111010001001011010011011
111010100111101111110011001101000101101001001010
1001100100101011001001110001011010001111011011111111100100110011010101011000
1001111111111001011000110011010010111010000
0010100011110100011110101111001110001101101100011000000100100110001101111001
\N
00100111100101110101000100101100111110101111010011011111100
0101101011001011010001100111101110100011101011110101000001011010100100010011001111001111
10001110100111110101001001010111101011111110
1100110100000010111010100
01
1100010010
1010001110011001101000111001001101001111101000101110110011100001000100111110110111001000001111101001
000110000111001111110000010010001100001010111110100111011
1100100111100111010010110001111111101001001011100111010000101010000011110011100100111101100
10111000000010111000111010000100111110000110100010111001011010111001011010100001001101101000
11010100110100011001110111011110101011011101010110000110101111010101
0001110
0001010001111010010110111111101011110011110010010111110011001011000111001101101011101000100110
11000001110001111100001110100110101111010000101011100000001000000000100011001001100011001011
01101111010100000
00000110011100001100001000000000001110111001001001001110000110101001011010110100
00111001000011111001111111001011000
101010000111001010001010000000011001111101111110000011111000101101101
011111110011011011001001001111111001101100111000001101011110000100110000001
001011000
001000010011100101100101011
\N
010101111010011101110100100110
0100100100100000111110111101
000000010000110111100011100001101110111011110000001
11111111001011011110010
101000101000000110010010111001000001111001010100101001111000001100110111000000
011010010100000101001011010001110010000010101111001101100011110110100010100111000000011
11111100110010001100000100111100010000010
01011010101000000100000111011011100100111101001001
010111101111000001101001001110001000100001111101011010011011100101110000101110
100000101
10011111001000001001101111010101111010
0010111011001001010100100000100
0001010100010101111011111110101101010000
011110001010001011101000111010011110000011000010001110010101010001111101
11000100101010101101110001101111000000001100111010
0001100100011101010101001100110110011111110010100100110
10011000011000101111010101001000001000001010111001100101110101011101111001101000111111100101100100
1000101111100110110111000001
11000101110110001001110000010010000001111010101101011001100000101011101100000101
100000111010100011010010010101
110011110100110101011000101011110101010110001101001000110001110110110100010010001111111100100011
0111111011111001010010101100110100011111110110001111001000001001001101100100111101100011010100
1110010101001111101010011000000100100100011101011101110001110001101110110010100111110010001000011
00110010100100010100111001100111000101100010000101110111101001
1011001011010000101001100001111101010001101110010111100001001
1101110100101110000001010110100100111111010100001011000110110101101000110101011110010
111110000001000100101001011011101111010000111100011011111001101000011101110110001
10000100101001010010101011100111101011000100011001110100000100010000110100100000001101
0010110000100010101111000010010011011011100011110110110100101001100010000
010011000110111110100000001110001001110011
0101010100111111011000100100100110101001100100110
1001100111000101100010000101110111101001101000001001011001100
1101011110011000000100011011000110010111001101000111100111111001
11000000000001001111011010010111001100010110100010001000
000111001011000110001111111100110
11111101111010101110110101101111110001110011100101101100000110011001000101010011001011
110100111111100110100111010000101111111100110100000010010010000100010001
00101000101000100000
101100111100011100000110010
011011101011101011110100111110010011100000101010100010
011000010000111010110111110010010010011101000101101001001001100001101011
1000101010001111001000010110110100000110111001010000101001000010101100101011100001011111101000000
000111100100000100100110110010011110110001101010001110000010111101011011000100000111111101
\N
101000011110101111111000000101111001001001
0011010000001011111010100101010000001110101001010111010111011000101011101101000111101
001111111110000101110100100101011010
0101
110011011101000001011010
0110001111010110011111101010100110000011001110110000011100000110
100101010110111011001001110000011010111111101110110001010111010001111000001010010
00001110010011111100111101000100011010
11110101011010000100111011100101010110011010010011110100101100100001100111100010100001
1001011100000110100101111010010011001001111000010100101101010001000001010111011110010001111100
110111110010011111001010101101011001001000111100111011001010101110000100110011001110101011000101
1101001011111100111000111001
0110100000110010100100010100111001100111000101100010000101
10010110110110011110011110100110101011000101011110101010110001101001000110001110110110100
0001000001011001101101101100101101111000110110000010011011101011101101100
00101011110110100011011111001001111100101010110101100100100011110011101100101010111000010011001
011011100110000
0001001011011111010010100101010001101101010011101001110000110
0101110010100010000001011010110010001010010010001010000100110
00
111100111111000110011001101101010
\N
111011101
0011110001100
110100000000000101000111101011010100111100011110010011
100000001100100100000010101111000111000011100000100001
010101100011010010001100011101101101000100100011111111001000110101011100000111000000010
011000000001110110111110001101100
0100001110010010110
10
000011011100101100000010011010111010101010001111010010101111100
1000
001101000101110010000011
10110010000100010010011100001100110100000000000101000
0010110101011100111111000000011000001111010011100110000010001000101110001
1010011100110111010000101111010110110110100001000010100101
1000110110010010010011111110111000100001111111001010100001
011011001110000010100001001011001010110001000011110100010010101101101011011000100110111
11010001011010101101
111110101100110010111010101110010100011001010011001111101110110010100
000111101111101110110000110000000100101101001000010010001110110110001111001000
00111101000000010111
1101001011000101000101011111111000000011
110111011111000110011011100011101100000100111100001010101011111111000100111011011001110110010010011
000010110011011011011001011011110001101100000100110111010111011011001
010011110001100001101110100011100111010111101110010
11110110110011110001101100101110000001
01001001101000101110011011010111110001100100110001001000111000101010111001010000111100
1010111010101110010111
1111110000000001010010000011
110011111001001001000110111000110001100101100110100011010100111101
101110111101001101000001001011001100001101001000001101
01
11001110110100000000000010101001100101100000111100000110101101000010
1100000110110101111001101
100
00011110101110001001
111110111010100111100011010011001001100111100110010
0100111110000110100010111001011010111001011010100001001101101000000011000
101111100110010110001110011011010111010001001100101001110110010111011111101
00010011010111001111101
10100011100001000111010110111
0111010010111111101111110111101010101101010
10111111011
10001011010
1000011111010110100110111001011100001011101100010001101000111110
010100100111101110110111001110110011110
000110101001111100110000101001111011101011001100000011010011
000111
10010001110110110001111001000101000101111010100110011101111101111100100011
01111101010001110011001101000111001001101001111101000101110110011100001000100
10100001110001
0101100000100001100001010011001010010001110000110010000110010111101110011010110100011
11101000100101010010111110
11010010110111011011101110100110000000110100001101100101101010000111110011110100100110100011
1100000000100100010110110101001101011010010100000111111110011100111011111111010111
101110111001100111010001111001101110101001101100010000001000001100010101001000001101100110110000001
01001011100100010001011
0001010111010000000111010101001110101110000011010100
110000111000011001001110111100001111111111001111101
00110001000011100
0110110011
0011110111110000001011011101101100000
0100011101011000011011100100110110011111000011001010010001011101111011001101100101001
10
1
00111101111000100110101010001100001001111010101110000001001001100101010111111001010111001001
011010011000110110100111010110000000111111111100100010111111101110101010111000000
111000000100010100001111001011011001101011000101111001000100001010011101
10100011110100101101111111010111100111100100101111100110010110001110011
100100000011110111000001011100100000101001000101000001001100010
0000100100010100101010010010110111011010110011110001101001010110100101
0101100011110010101111110100001110010011101100000000011
1001100111100100010110011111101011111101011011010011001101
1100001000111101010001100111111101111010001100000
010001111001101100100000111010110
01000011001110010011001010000
0111011100101010110011010010011110100101100100001100111100010100001000111
011110111110000001011011101101100000
101011010101101100010111111011011101110001011110010001001000001101001111001100011100011001000010
0001100011101101101000100100011111111001000110101011100000111000000010000011
11000111111000111110000000000110110101001110110100101011010101010101000010000000110
1011
1110110101110110010001011011110110000110000111111001011110010011010111001000111010010001
110101100010110101000
10110011011101101001111011110001001101010100011000010011110101011100000010010011001010101
00011111100100010
00
01010010000001011010101000000100000111
11010010110110101001110011011101010110101100010111011000100111
10101000110011010000011000101110011000111110011111101111100110100011000011111001100100
11111100001000011001101110110101101010111
10110011000111011
01000111100111
0000110101010000010111111101101001011100111111101011010111001111110111000100010011001101011011100
10011100010110000111101101111101100100101010000010011010111110101010110101011100011010011101000010
110111110101000001111011110010010010010111110000100001001111001010010010010100110100
001100000000
10000101101110011110001000000110
00101010101001101111000
011000111001010000111101
0110100000111000001111000101010000110110100101101100
1011100101010110011010010011110100101100100001100111100010100001000111
