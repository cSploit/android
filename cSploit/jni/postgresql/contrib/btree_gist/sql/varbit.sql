-- varbit check

CREATE TABLE varbittmp (a varbit);

\copy varbittmp from 'data/varbit.data'

SET enable_seqscan=on;

SELECT count(*) FROM varbittmp WHERE a <   '1110100111010';

SELECT count(*) FROM varbittmp WHERE a <=  '1110100111010';

SELECT count(*) FROM varbittmp WHERE a  =  '1110100111010';

SELECT count(*) FROM varbittmp WHERE a >=  '1110100111010';

SELECT count(*) FROM varbittmp WHERE a >   '1110100111010';

CREATE INDEX varbitidx ON varbittmp USING GIST ( a );

SET enable_seqscan=off;

SELECT count(*) FROM varbittmp WHERE a <   '1110100111010'::varbit;

SELECT count(*) FROM varbittmp WHERE a <=  '1110100111010'::varbit;

SELECT count(*) FROM varbittmp WHERE a  =  '1110100111010'::varbit;

SELECT count(*) FROM varbittmp WHERE a >=  '1110100111010'::varbit;

SELECT count(*) FROM varbittmp WHERE a >   '1110100111010'::varbit;
