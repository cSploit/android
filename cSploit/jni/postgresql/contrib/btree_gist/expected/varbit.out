-- varbit check
CREATE TABLE varbittmp (a varbit);
\copy varbittmp from 'data/varbit.data'
SET enable_seqscan=on;
SELECT count(*) FROM varbittmp WHERE a <   '1110100111010';
 count 
-------
   549
(1 row)

SELECT count(*) FROM varbittmp WHERE a <=  '1110100111010';
 count 
-------
   550
(1 row)

SELECT count(*) FROM varbittmp WHERE a  =  '1110100111010';
 count 
-------
     1
(1 row)

SELECT count(*) FROM varbittmp WHERE a >=  '1110100111010';
 count 
-------
    51
(1 row)

SELECT count(*) FROM varbittmp WHERE a >   '1110100111010';
 count 
-------
    50
(1 row)

CREATE INDEX varbitidx ON varbittmp USING GIST ( a );
SET enable_seqscan=off;
SELECT count(*) FROM varbittmp WHERE a <   '1110100111010'::varbit;
 count 
-------
   549
(1 row)

SELECT count(*) FROM varbittmp WHERE a <=  '1110100111010'::varbit;
 count 
-------
   550
(1 row)

SELECT count(*) FROM varbittmp WHERE a  =  '1110100111010'::varbit;
 count 
-------
     1
(1 row)

SELECT count(*) FROM varbittmp WHERE a >=  '1110100111010'::varbit;
 count 
-------
    51
(1 row)

SELECT count(*) FROM varbittmp WHERE a >   '1110100111010'::varbit;
 count 
-------
    50
(1 row)

